docs/RunningApiTests-README.md

```diff
@@ -0,0 +1,47 @@
+# API Test Execution:
+
+## Command: 
+CONFIG=./configs/<apiConfig.properties> TAG=<ScenarioTag> PLATFORM=api ./gradlew run
+
+## API Config.properties File Params:
+
+```
+RUNNER=distribute 
+FRAMEWORK=cucumber
+RUNNER_LEVEL=methods
+APP_NAME=<API SCENARIO APP NAME>
+ENVIRONMENT_CONFIG_FILE=./src/test/resources/environments.json
+IS_VISUAL=false
+LOG_DIR=target
+LOG_PROPERTIES_FILE=./src/test/resources/log4j.properties
+PARALLEL=<parellel count>
+PLATFORM=api
+PROXY_KEY=HTTP_PROXY
+REPORT_PORTAL_FILE=src/test/resources/reportportal.properties
+RUN_IN_CI=<true/false>
+TARGET_ENVIRONMENT=<sit/eat/prod>
+LAUNCH_NAME_SUFFIX= on <environment> Environment
+TEST_DATA_FILE=./src/test/resources/testData.json
+CLEANUP_DEVICE_BEFORE_STARTING_EXECUTION=false
+```
+
+## Sample API scenario Example:
```

Comment:
Add a sample API Workflow scenario example as well

Reply:
added to readme